MBBSEmu/DOS/Structs/DTAStruct.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
```

Comment:
is this part of another branch? Doesn't seem to be used at all

Reply:
It's a namespace within the .NET Framework, holds the `Array` class used for the data copying